packages/core/chrome/core-chrome-browser/src/project_navigation.ts

```diff
@@ -37,7 +40,11 @@ export type AppId =
   | MlApp
   | ManagementApp
   | SearchApp
-  | ObservabilityApp;
+  | ObservabilityApp
+  | ObservabilityApp
```

Comment:
NIT: this is duplicated

Reply:
I had it fixed locally, forgot to push ... Done in https://github.com/elastic/kibana/pull/178861/commits/466f84056ba99ffffdfcdbb288f44676baf1219c